### ISSUE TITLE
Test failing after merge

### ISSUE THREAD

**Issue by carloslfu** [carloslfu/xstate-router#4]
After merging #3 , there is a test failing, this needs investigation in order to be able to release a new version to NPM.

Test: XStateRouter › When enter a state having a parameterized route, the route sho
uld reflect the current param value. - [line 180](https://github.com/carloslfu/xstate-router/blob/master/src/index.spec.tsx#L180)

```javascript
    expect(received).toBe(expected) // Object.is equality

    Expected: "/substate/817/c"
    Received: "/substate/undefined/c"

      178 |     const { getByTestId } = renderWithRouter(App, { route: '/about' })
      179 |     fireEvent.click(getByTestId('go-substate-c'))
    > 180 |     expect(getByTestId('location-display').textContent).toBe('/substate/817/c')
          |                                                         ^
      181 |   })
```

**Comment by stephanpelikan** [comment]
A clean checkout of 'master' and doing a `npm install` and `npm test` results in

```
$ npm test

> xstate-router@0.2.0 test /var/tmp/xstate-router
> jest

 PASS  src/index.spec.tsx
  XStateRouter
    ✓ When enter a route, should update the state (78ms)
    ✓ When enter a route and the machine enters to a routable substate, should update the route (20ms)
    ✓ When enter a routable state, should update the route (39ms)
    ✓ When go back in history, should update state acordinglly (17ms)
    ✓ When enter a routable state, should be able to redirect state update (11ms)
    ✓ When enter a routable parameterized state, match should contain parameter value (6ms)
    ✓ When enter a routable state, should be able to stop state update (7ms)
    ✓ When enter a substate of a routable state from other routable state, should update the route (7ms)
    ✓ When enter a state having a parameterized route, the route should reflect the current param value (7ms)

Test Suites: 1 passed, 1 total
Tests:       9 passed, 9 total
Snapshots:   0 total
Time:        3.942s
Ran all test suites.
```
It seems like you are testing the old status of master using the new test. Can you confirm my result by using a clean clone?

**Comment by carloslfu** [comment]
I solved it by updating dependencies and, just released v0.3.0. Thanks, @stephanpelikan.
